### PR TITLE
October 2021 sprint

### PR DESCRIPTION
**Pull request by arunkannawadi**

### CODE REVIEW

**Comment by humnaawan** [comment]
thanks @arunkannawadi for the updates! a few comments:


1.  we are missing the hub-specific channels: [#desc-sprint2021-apc-logistics](https://join.slack.com/share/enQtMjYyMDUxOTA5NTA3OC0xZDYzODNkNjdlODE2MjRhMzM3NzJkOWNhOTkwNmFlY2FlOTM3YjVkZTU0MThjYjkwNzViYjM4YThkZTgyMzQ1), [#desc-sprint2021-princeton-logistics](https://join.slack.com/share/enQtMjYyNzI1NjMwODcyMi1jMzU5NDg4YmEyNjA1YTE0MmJiZjVlNjEwZWU5ZTVjOWNhNzI4OWRlNGU2NWE3MWU4ZWFiNDQ0MzYyMjdhNzMy), [#desc-sprint2021-uiuc-logistics](https://join.slack.com/share/enQtMjYyNzI1OTU1MjY5MC1lNTE2Y2MxZDg2NzQ0MTYzZjI5NTE3MTkzYTQ3YzQyZGU5OTJmZTE1ZDc1N2FkNjFjZGZiZDU0Yzk0MDRkZjI4). can you please add these when you mention the corresponding channel to ones hub.
2. is it true that only DP0 delegates can access the Rubin Science Platform? if that is the case, then it really doesn't make sense for us to advertise it.
3. the checklist is missing IN2P3 info. You can borrow text from the Logistics page:
- As an LSST member, you can request the setup of an IN2P3 account, you will find all the necessary information [Getting an IN2P3 Account](https://confluence.slac.stanford.edu/display/LSSTDESC/Getting+an+IN2P3+Account) and here: https://doc.cc.in2p3.fr/en/Getting-started/start.html#computing-account
- Pay attention to requesting your account before the beginning of the week, as it’s quick but might still need 24 to 48h to be up and running.
4. it would help to have main bullets under `Computing resources etc.` be "NERSC", "IN2P3", "Rubin Science Platform" (assuming we keep that; see comment 2 above), "DC2", since currently, it all seems too much?
5. maybe it'd be good to add COVID-19 policy info for each hub? You can get the text from the Logistics page, or just say "Please see the Logistics page <linked> for COVID policy for the hub you are attending."?

thanks!

**Comment by arunkannawadi** [comment]
- [x] we are missing the hub-specific channels: #desc-sprint2021-apc-logistics, #desc-sprint2021-princeton-logistics, #desc-sprint2021-uiuc-logistics. can you please add these when you mention the corresponding channel to ones hub.
- [x] is it true that only DP0 delegates can access the Rubin Science Platform? if that is the case, then it really doesn't make sense for us to advertise it.
- [x] the checklist is missing IN2P3 info. You can borrow text from the Logistics page:
As an LSST member, you can request the setup of an IN2P3 account, you will find all the necessary information Getting an IN2P3 Account and here: https://doc.cc.in2p3.fr/en/Getting-started/start.html#computing-account
Pay attention to requesting your account before the beginning of the week, as it’s quick but might still need 24 to 48h to be up and running.
- [x] it would help to have main bullets under Computing resources etc. be "NERSC", "IN2P3", "Rubin Science Platform" (assuming we keep that; see comment 2 above), "DC2", since currently, it all seems too much?
- [x] maybe it'd be good to add COVID-19 policy info for each hub? You can get the text from the Logistics page, or just say "Please see the Logistics page for COVID policy for the hub you are attending."?

**Comment by arunkannawadi** [comment]
@yymao I am unable to request a reviewer on this PR, but this is good to be merged after LOC review

**Comment by yymao** [comment]
@arunkannawadi I just gave you write access to this repo

File: index-july-2021-virtual.html [hunk 1]
```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+     <title>Checklist for the July 2021 LSST DESC Virtual Collaboration Meeting</title>
```
Comment (humnaawan) [inline index-july-2021-virtual.html:5]: i think this needs to be updated?

**Review by humnaawan — APPROVED** [review]

**Pull request merged**